main.rb

```diff
@@ -0,0 +1,53 @@
+require_relative './school/actions/actions'
+
+class App
+  def initialize()
+    @actions = Actions.new
+  end
+
+  def init
+    main while @actions.keep_going
+  end
+
+  def do_action(option)
+    case option
+    when '1'
+      @actions.list_books
+    when '2'
+      @actions.list_people
```

Comment:
- [ ] Listing people shows unwanted repeating "name: Unkown, id, age" before each person data, kindly fix it so it matches the requirements.
- [ ] There should be no parent permission in any of the teacher's entries, instead, it should be the teacher's specialization.
![image](https://user-images.githubusercontent.com/33767951/156506012-50e7aba4-0729-48a7-b554-7f68e39b7c4b.png)